cranelift/assembler-x64/meta/src/instructions/avg.rs

```diff
@@ -0,0 +1,16 @@
+use crate::dsl::{Feature::*, Inst, Location::*, VexLength::*};
+use crate::dsl::{align, fmt, inst, r, rex, vex, w};
+
+#[rustfmt::skip] // Keeps instructions on a single line.
+pub fn list() -> Vec<Inst> {
+    vec![
+        inst("pavgb", fmt("A", [w(xmm1), r(align(xmm_m128))]), rex([0x66, 0x0F, 0xE0]), _64b | compat | sse2),
+        inst("pavgw", fmt("A", [w(xmm1), r(align(xmm_m128))]), rex([0x66, 0x0F, 0xE3]), _64b | compat | sse2),
```

Comment:
I think these need to be `rw(xmm1)` along with updating the ISLE below to pass tests?

Reply:
@Alex .. hadn't seen your comment until now, but I just discovered this while working on pcmp. Will try this out and see if it resolves.